app/locale/et_EE/Sendsmaily_Sync.csv

```diff
@@ -25,3 +25,4 @@
 "The configuration has been saved.", "Sätted salvestatud"
 "Module is not activated!", "Smaily moodul ei ole aktiveeritud"
 "Could not validate credentials! Your current input is saved, but the module is still disabled. Please try again!", "Ei õnnestunud kasutajat valideerida! Praegused andmed on salvestatud, kuid moodul on endiselt deaktiivne. Palun proovi uuesti!"
+"General Settings","Üldised Sätted"
```

Comment:
"Sätted" should start with a lowercase letter.

Reply:
![image](https://user-images.githubusercontent.com/22918359/69225427-0fb17500-0b87-11ea-96d1-43a41476f6ba.png)
Currently, I have kept the label elements of sections with uppercase first letters. Should I change that?

Reply:
Please change it.

Reply:
Changed in https://github.com/sendsmaily/Sendsmaily-Sync-for-Magento/pull/6/commits/c6c7c590e2bbb87151c4c5e74b33a5aee7ba4d8c